Extension/CHANGELOG.md

```diff
@@ -1,23 +1,43 @@
 # C/C++ for Visual Studio Code Change Log
 
-## Version 0.17.8: August 16, 2018
-* Fix attach to process for systems without `bash` by using `sh` instead. [#569](https://github.com/Microsoft/vscode-cpptools/issues/569)
-  * Andy Neff (@andyneff) [PR 2340](https://github.com/Microsoft/vscode-cpptools/pull/2340)
-* Fix IntelliSense crash after hover or completion with `_Complex` types. [#689](https://github.com/Microsoft/vscode-cpptools/issues/689), [#1112](https://github.com/Microsoft/vscode-cpptools/issues/1112)
+## Version 0.18.0: August 16, 2018
+### New Features
+* Add `C_Cpp.disableLanguageService` setting (for users who only want the debugger features). [#785](https://github.com/Microsoft/vscode-cpptools/issues/785)
```

Comment:
this will need to be updated based on the changes requested to the active PR, but you can update it in that PR.

Reply:
It's updated.